Detectors/EMCAL/simulation/src/DMAOutputStream.cxx

```diff
@@ -30,7 +30,19 @@ void DMAOutputStream::open()
   }
 }
 
-void DMAOutputStream::writeData(RawHeader header, gsl::span<char> buffer)
+void DMAOutputStream::writeSingleHeader(RawHeader header) 
+{
+  if (!mInitialized)
+    open();
+  std::vector<char> outputpage(sizeof(RawHeader));
+  RawHeader *outputheader = reinterpret_cast<RawHeader *>(outputpage.data());
+  *outputheader = header;
+  header.memorySize = sizeof(RawHeader);
```

Comment:
These assignments have no any effect, I guess you wanted to have 
````
outputheader.memorySize = sizeof(RawHeader); 
outputheader.offsetToNext = sizeof(RawHeader); 
````

Reply:
Yes, this was a bug. Thanks for spotting!